src/screens/DetailedViewScreen.tsx

```diff
@@ -1,75 +1,83 @@
+	setPage: (newPage: string) => void;
 }
 
+const BodyText = (props: TextProps) => <Text style={styles.text} {...props} />;
+
 const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlaceProps) => {
-	const [placeDetails, setPlaceDetails] = useState<{ placeDetails: PlaceDetailsResponseData }>();
+	const { placeDetails } = useFetchPlace({ placeID });
 
 	const { location } = useLocation();
+
 	const userCoords = {
 		latitude: location?.coords.latitude,
 		longitude: location?.coords.longitude,
 	};
 
-	const placeCoord = {
-		latitude: placeDetails?.placeDetails.result.geometry?.location.lat,
-		longitude: placeDetails?.placeDetails.result.geometry?.location.lng,
-	};
-	const placeCoords = { latitude: placeCoord.latitude, longitude: placeCoord.longitude };
-
-	const distanceInMi = useMemo<string | undefined>(
-		() => computeDistanceMi(userCoords, placeCoords)?.toPrecision(2),
-		[userCoords, placeCoords]
-	);
-
-	const getPlaceDetails = async (placeID: string) => {
-		const result = await axios.get<{ placeDetails: PlaceDetailsResponseData }>(
-			`${SERVER_BASE_URL}/getPlaceDetails/${placeID}`
-		);
-		setPlaceDetails(result.data);
-	};
+	if (placeDetails) {
+		const place = placeDetails.placeDetails.result;
 
-	useEffect(() => {
-		(async () => {
-			if (placeID) {
-				await getPlaceDetails(placeID);
-			}
-		})();
-	}, [placeID]);
+		const placeCoord = {
+			latitude: place.geometry?.location.lat,
+			longitude: place.geometry?.location.lng,
+		};
+		const placeCoords = { latitude: placeCoord.latitude, longitude: placeCoord.longitude };
+		const distanceInMi = computeDistanceMi(userCoords, placeCoords)?.toPrecision(2);
 
-	if (placeDetails) {
 		return (
 			<View style={styles.border}>
-				<Text ellipsizeMode="tail" numberOfLines={2} style={styles.title}>
-					{placeDetails?.placeDetails.result.name}
-				</Text>
-				<Text ellipsizeMode="tail" numberOfLines={2} style={styles.text}>
-					{placeDetails?.placeDetails.result.formatted_address}
-				</Text>
-				<Text
-					accessibilityLabel={distanceInMi ? `${distanceInMi} miles away` : ""}
-					style={styles.text}
-				>
-					{distanceInMi ? `${distanceInMi} mi away` : ""}
-				</Text>
-				<Button
-					style={styles.homeBtn}
-					onPress={setPage}
-					accessibilityLabel="Return to Home Page"
-					text="Go Home"
+				<PlaceImage
+					photoref={place.photos?.[0]?.photo_reference}
+					placeName={place.name}
+					style={{
+						width: "100%",
+						height: "40%",
+					}}
 				/>
+				<View style={styles.content}>
+					<Text ellipsizeMode="tail" numberOfLines={2} style={styles.title}>
+						{place.name}
+					</Text>
+					<BodyText ellipsizeMode="tail" numberOfLines={2}>
+						{place.formatted_address}
+					</BodyText>
+					<BodyText accessibilityLabel={distanceInMi ? `${distanceInMi} miles away` : ""}>
+						{distanceInMi ? `${distanceInMi} mi away` : ""}
+					</BodyText>
+					<BodyText>
+						Accessibility Rating:{" "}
+						{getAverageA11yRating({
```

Comment:
I noticed that the place list uses `Math.round(getAverageA11yRating(place) * 2) / 2`, while we use this here... I feel like we should change one or the other to be consistent